package.json

```diff
@@ -367,7 +367,7 @@
     "unset-value": "2.0.1",
     "webpack-dev-middleware": "^5.3.4",
     "nanoid": "^3.3.8",
-    "path-to-regexp": "^0.1.12"
```

Comment:
this forces all `path-to-regexp` to resolve to a single version which would break libraries that require v6+ since the exports have changed.